treeherder/perf/models.py

```diff
@@ -545,14 +545,14 @@ class BackfillRecord(models.Model):
     PRELIMINARY = 0
     READY_FOR_PROCESSING = 1
     BACKFILLED = 2
-    FINISHED = 3
+    SUCCESSFUL = 3
     FAILED = 4
 
     STATUSES = (
         (PRELIMINARY, 'Preliminary'),
         (READY_FOR_PROCESSING, 'Ready for processing'),
         (BACKFILLED, 'Backfilled'),
-        (FINISHED, 'Finished'),
+        (SUCCESSFUL, 'Successful'),
```

Comment:
Doesn't this need a migration script?

Reply:
I checked this locally. Indeed, this requires a new migration script, otherwise this will blow up when merging to master.